lib/uplink/bucket_attrs_test.go

```diff
@@ -59,6 +60,73 @@ func simpleEncryptionAccess(encKey string) (access EncryptionAccess) {
 	return access
 }
 
+func createPartnerID() (partnerID string, err error) {
+	id, err := uuid.New()
+	if err != nil {
+		return partnerID, err
+	}
+	return id.String(), err
+}
+
+// check that partner bucket attributes are stored and retrieved correctly.
+func TestPartnerBucketAttrs(t *testing.T) {
+	var (
+		access          = simpleEncryptionAccess("voxmachina")
```

Comment:
you don't have to create such custom bucket config. it doesn't matter for attribution testing.

Reply:
updated the test case